services/movie.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[id] = None
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+
+    if genres_ids is not None and actors_ids is not None:
+        return queryset.filter(
+            genres__id__in=genres_ids
+        ).filter(
+            actors__id__in=actors_ids
+        )
+
+    if genres_ids is not None:
+        return queryset.filter(
+            genres__id__in=genres_ids
+        )
+
+    if actors_ids is not None:
+        return queryset.filter(
+            actors__id__in=actors_ids
+        )
```

Comment:
code can be simplified, no need for several return statements.

queryset can be filtered based on genres_ids and actors_ids independently, and only if they are provided.